src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -144,6 +152,12 @@ CHIP_ERROR writeAdminsIntoFabricsListAttribute()
         err = CHIP_ERROR_PERSISTED_STORAGE_FAILED;
     }
 
+    if (err == CHIP_NO_ERROR && writeCommissionedFabricsAttribute(fabricIndex) != EMBER_ZCL_STATUS_SUCCESS)
```

Comment:
`fabricIndex` presumably should not be an `int32_t`...

Reply:
fixed it